website/content/api-docs/auth/saml.mdx

```diff
@@ -0,0 +1,427 @@
+---
+layout: api
+page_title: SAML - Auth Methods - HTTP API
+description: |-
+  This is the API documentation for the Vault SAML auth method.
+---
+
+# SAML Auth Method (API)
```

Comment:
We should use sentence case for all titles

```suggestion
# SAML auth method (API)
```

Reply:
Thanks! Done in https://github.com/hashicorp/vault/pull/23183/commits/b611152c9b94909872a6cc31040ff726301ec63c.